day_6/exercises/burrito.js

```diff
@@ -16,12 +16,36 @@ class Burrito {
     this.base = base;
     this.toppings = toppings;
   }
+  add_topping(topping) {
+    this.toppings.push(topping);
+  }
+  remove_topping(topping) {
+    this.toppings.pop(topping);
```

Comment:
Does pop take an argument? this is something to look for an consider when looking at mdn